src/radical/pilot/agent/radical-pilot-agent-multicore.py

```diff
@@ -152,12 +152,17 @@ class UpdateWorker
 import traceback
 import subprocess
 import multiprocessing
+import commands
+import json
+import urllib2 as ul
 
 import saga                as rs
 import radical.utils       as ru
 import radical.pilot       as rp
 import radical.pilot.utils as rpu
 
+from datetime import datetime
+
```

Comment:
yes: http://stackoverflow.com/questions/89228/calling-an-external-command-in-python